src/client/datascience/baseJupyterSession.ts

```diff
@@ -150,58 +155,15 @@ export abstract class BaseJupyterSession implements IJupyterSession {
         }
         return this.session.kernel?.requestKernelInfo();
     }
-    public async changeKernel(
```

Comment:
Moved this, this is not required for raw kernels, hence moved into JupyterSession